config/secrets.yml

```diff
@@ -13,6 +13,7 @@
 development: &development
   secret_key_base: e369575030a3ac3c396ccb152187c43e61e03721a910e5a90a1153c0f06b04c2a2b750ffb0020152a6580c261cb86ae11936f0ea2deec6380012b33d35b7fd3c
   email_recipients_interceptors: <%= ENV['EMAIL_RECIPIENTS_INTERCEPTOR'] %>
+  max_requests_per_second: <%= ENV.fetch("MAX_REQUESTS_PER_SECOND", 5) %>
```

Comment:
This is going to run on development too? i don't think we need these restrictions here